ts/components/screens/BaseScreenComponent/index.tsx

```diff
@@ -199,20 +95,14 @@ const BaseScreenComponentFC = React.forwardRef<ReactNode, Props>(
               })
             );
           };
-        case ToolEnum.instabug:
-          // TODO: remove instabug
-          return () => showHelp();
-        case ToolEnum.none:
-        case ToolEnum.web:
-          return undefined;
```

Comment:
Here I would feel more comfortable by leaving all cases (no `default`): when we remove or add some other cases, the code doesn't compile since we have not implemented all required cases. 
i.e if we will introduce `newAssistanceTool2022` is it ok to return `undefined` for that scenario? in the other hand the code will raise an error saying and we know where to update the logic

<img width="376" alt="Schermata 2022-03-31 alle 11 32 40" src="https://user-images.githubusercontent.com/822471/161024581-de95e6c5-3e76-4048-a2af-cee37ca879f2.png">

Reply:
Yes, safer for me too. Thank you. Modified in [67ab20c](https://github.com/pagopa/io-app/pull/3867/commits/67ab20c3aac1b8207c481f0374daf50d6920bf8f)